pkg/apiserver/server.go

```diff
@@ -70,20 +70,25 @@ func newEngine() *gin.Engine {
 		v.RegisterValidation("RequiredFieldEqual", apivalidator.RequiredFieldEqualValid, true)
 	}
 
-	moveToUIRoot := func(c *gin.Context) {
-		c.Redirect(http.StatusMovedPermanently, "/dashboard")
-	}
-
-	r.GET("/", moveToUIRoot)
 	ui := uiserver.AssetsFS()
 	if ui != nil {
-		newDashboardRouter(r, ui)
+		r.GET("/", func(c *gin.Context) {
+			c.FileFromFS("/", ui)
+		})
+		r.GET("/:foo/*bar", func(c *gin.Context) {
+			c.FileFromFS("/", ui)
+		})
```

Comment:
It seems these lines of codes are unnecessary. Only `renderStatic` is the useful func.

Reply:
Not. Because the production build of the UI only has one entry, `index.html`, it's a single page application. So it's needed to redirect all requests to root.